package.json

```diff
@@ -2,7 +2,7 @@
   "name": "custom-server-typescript",
   "version": "1.0.0",
   "scripts": {
-    "dev": "next dev",
+    "dev": " BRANCH=teste next dev",
```

Comment:
'teste' or 'test'?

Reply:
oops, this was just for testing the environment variables, forgot about it.